### ISSUE TITLE
RS1024 when calling RuntimeHelpers.GetHashCode

### ISSUE THREAD

**Issue by YairHalberstadt** [dotnet/roslyn#49386]
**Version Used**: Microsoft.CodeAnalysis.Analyzers 3.3.1

**Steps to Reproduce**:

```csharp
RuntimeHelpers.GetHashCode((ISymbol)null!);
```

**Expected Behavior**: No error as RuntimeHelpers.GetHashCode is a specialized hashcode function for checking if two things are the exact same instance. When using it you are explicitly opting out of using normal equality.

**Actual Behavior**: RS1024 compare symbols correctly

**Comment by FelixDamrau** [comment]
Is there any progress on this issue? I am experiencing the exact same issue.
Maybe I'm just doing things wrong...

**Comment by FelixDamrau** [comment]
Is there any additional info I can provide, so this issue can get solved?

**Comment by CyrusNajmabadi** [comment]
@FelixDamrau The issue is in the backlog. If you're interested, you could create a pr to fix things. For now though, the workaround would be too suppress that diagnostic.

**Comment by FelixDamrau** [comment]
Thanks for the quick reply.
I did not grasp the meaning of: *@[jaredpar](https://github.com/jaredpar) jaredpar added this to the [Backlog](https://github.com/dotnet/roslyn/milestone/2) milestone [on Nov 17, 2020](https://github.com/dotnet/roslyn/issues/49386#event-4007749450)*.

My bad 🤦‍♀️
